apps/pie-storybook/stories/pie-lottie-player.stories.ts

```diff
@@ -0,0 +1,130 @@
+            control: {
+                type: 'select',
+                labels: animationSrcLabels,
+            },
+            options: [...animationSrcOptions, ''],
+            defaultValue: {
+                summary: defaultArgs.animationSrc,
+            },
+        },
+        aria: {
+            description: 'The ARIA settings used for the component.',
+            control: 'object',
+        },
+        loopDisabled: {
+            description: 'By the default animations loop, setting this prop as true will prevent such behaviour.',
+            control: 'boolean',
+            defaultValue: {
+                summary: defaultArgs.loopDisabled,
+            },
+        },
+        autoPlayDisabled: {
+            description: 'By default animations start playing as soons as its data is available, setting this prop as true will prevent such behaviour.',
+            control: 'boolean',
+            defaultValue: {
+                summary: defaultArgs.autoPlayDisabled,
+            },
+        },
+        speed: {
+            description: 'Determines the animation reproduction speed.\n1 is the regular speed, 2 is twice as fast.',
+            control: {
+                type: 'number',
+                step: 0.25,
+                min: 0.25,
+            },
+            defaultValue: {
+                summary: defaultArgs.speed,
+            },
+        },
+        direction: {
+            description: 'Sets the animation reproduction direction',
+            control: 'select',
+            options: directions,
+            defaultValue: {
+                summary: defaultArgs.direction,
+            },
+        },
+    },
+    args: defaultArgs,
+    parameters: {
+        design: {
+            type: 'figma',
+            url: '',
+        },
+    },
+};
+
+export default lottiePlayerStoryMeta;
+
+function handleStopAnimationClick () {
+    const player:PieLottiePlayer|null = document.querySelector('pie-lottie-player');
+    if (player) player.stop();
+}
+
+function handleStartAnimationClick () {
+    const player:PieLottiePlayer|null = document.querySelector('pie-lottie-player');
+    if (player) player.play();
+}
+
+const Template = ({
+    animationSrc,
+    aria,
+    loopDisabled,
+    autoPlayDisabled,
+    speed,
+    direction,
+}: LottiePlayerProps) => html`
+    <div style="display: flex; justify-content: center;">
+        <pie-button size="xsmall" @click=${handleStopAnimationClick}>Stop animation</pie-button>
+        <pie-button size="xsmall" @click=${handleStartAnimationClick}>Start animation</pie-button>
+    </div>
+    <pie-lottie-player
+        .animationSrc="${animationSrc === undefined ? animationSrcOptions[0] : animationSrc}"
+        .aria="${aria}"
+        ?loopDisabled="${loopDisabled}"
+        ?autoPlayDisabled="${autoPlayDisabled}"
+        .speed="${speed || 1}"
+        .direction="${direction || directions[0]}"
```

Comment:
Can we just rely on (and also test) the default prop behaviour? `1` and `"forward"` are the default values
```suggestion
        .speed="${speed}"
        .direction="${direction}"
```

Reply:
It bothered me a bit that setting 0 stops the animation :)

Reply:
That sounds expected to me 😄